### ISSUE TITLE
Module not found: Default condition should be last one

### ISSUE THREAD

**Issue by tuffstuff9** [karmaniverous/serify-deserify#22]
<img width="958" alt="image" src="https://github.com/karmaniverous/serify-deserify/assets/57072903/4c79db1c-d223-43f5-bde2-da5bf0f23fac">

**Comment by karmaniverous** [comment]
Sorry for the delay, just saw this. Maybe has something to do with [this](https://stackoverflow.com/questions/76127288/module-not-found-error-default-condition-should-be-last-one)?

Past my bedtime, I'll have a look at this tomorrow. Just wanted you to know I saw it.

**Comment by tuffstuff9** [comment]
> Sorry for the delay, just saw this. Maybe has something to do with [this](https://stackoverflow.com/questions/76127288/module-not-found-error-default-condition-should-be-last-one)?
> 
> Past my bedtime, I'll have a look at this tomorrow. Just wanted you to know I saw it.

Thank you for your response. I saw that, but I was under the impression that this is something that was to be fixed on the library's end, no?

**Comment by karmaniverous** [comment]
> Thank you for your response. I saw that, but I was under the impression that this is something that was to be fixed on the library's end, no?

Yes! And I think I've got it. Minor leftover from the TS refactor, looks like the exports were in the wrong order in `package.json`. Here's [a great reference](https://www.kravchyk.com/typescript-npm-package-json-exports/) if you're interested.

I just published a trial version at `@karmaniverous/serify-deserify@next`. Would you please give that one a shot and let me know how it goes?

**Comment by tuffstuff9** [comment]
Works great - thanks! Also, just some feedback on the docs:

When I was looking to use the library, I quickly scanned the docs and didn't see a quick redux implementation. And on the tests, it doesn't say how to import what we need from the library 

i.e. `import { createReduxMiddleware, serify } from '../';` 
instead of 
`import { createReduxMiddleware, defaultOptions } from '@karmaniverous/serify-deserify';`

I think if you just put this snippet in the docs, it would reduce a lot of friction for people who want to try the library out and get up and running quickly. 

```typescript
import {
	createReduxMiddleware,
	defaultOptions,
} from '@karmaniverous/serify-deserify';

// Create middleware.
const serifyMiddleware = createReduxMiddleware(defaultOptions);

// Construct slice.
const testSlice = createSlice({
  name: 'test',
  initialState,
  reducers: {
    setValue: (state, { payload }: PayloadAction<TestState['value']>) => {
      state.value = payload;
    },
  },
});

// Configure redux store.
const store = configureStore({
  reducer: combineReducers({
    test: testSlice.reducer,
  }),
  middleware: (getDefaultMiddleware) =>
    getDefaultMiddleware().concat(serifyMiddleware),
});
```

From the above snippet, the user understands that they can just pass in defaultOptions rather than custom options, as shown in the test. They also see where to import from, and finally, it prevents the user from searching through code that is not relevant to them.

**Comment by karmaniverous** [comment]
Awesome thank you! I've updated the `README` with your example & released `@karmaniverous/serify-deserify@2.02`.
